zschema/keys.py

```diff
@@ -168,6 +171,43 @@ def key_to_string(o):
         else:
             return o.to_string()
 
+    @staticmethod
+    def _handle_validation_exception(policy, e):
+        if policy == "error":
+            raise e
+        elif policy == "warn":
+            logging.warn(e.message)
+        elif policy == "ignore":
+            pass
+        else:
+            raise Exception("Invalid validation policy. Must be one of: error, warn, ignore")
+
+    @staticmethod
+    def _validate_policy(name, policy):
```

Comment:
Reading through the code, I'd initially assumed `validate_policy` validates that the policy was being followed.

What about `validation_policy_enum(policy, name=None)`, by analogy to e.g. `int()`? (or `validation_policy_id` or just `validation_policy` etc)

Then you could also use `type=validation_policy_enum` in the `argparse`.